pkg/trace/agent/agent.go

```diff
@@ -56,6 +56,9 @@ type Agent struct {
 	obfuscator     *obfuscate.Obfuscator
 	cardObfuscator *ccObfuscator
 
+	// ProcessSpan will be called on all spans, if non-nil.
+	ProcessSpan func(*pb.Span)
```

Comment:
Sorry to circle around here. We are using the word "process" a lot in our code. Can you please do one last change and rename this to `ModifySpan` or `AlterSpan` just to avoid using that word even more?

Reply:
sure!